backend/core/src/listings/listings.service.spec.ts

```diff
@@ -306,14 +307,17 @@ describe("ListingsService", () => {
   })
 
   describe("ListingsService.list sorting", () => {
-    it("verify both queries have correct ORDER BY clauses", async () => {
+    it("verify the default ordering (no orderBy param)", async () => {
```

Comment:
nit: can we name this so it reads like a sentence starting with "it", so something like what you said elsewhere: "defaults to ordering by application due date when no orderby param is set"

Reply:
Thanks, done!